### PR TITLE
Support Nexus repos in helm_push

### LINKED ISSUE #24: helm_push: support Sonatype Nexus repositories
Hi!

Thank you for wrapping Helm in Bazel rules! There's one caveat that we've discovered: without altering the rule, it is impossible to upload built artifacts into the Helm repositories hosted by Sonatype Nexus. Most probably, that's happening due to Nexus having a separate upload method from one that usual Helm repos support. Nexus documentation claims [here](https://help.sonatype.com/repomanager3/formats/helm-repositories#HelmRepositories-UploadbyHTTPPOST) that one must use `--upload-file` to achieve said task:

```
curl -u <username>:<password> http://<host>:<port>/repository/<repository_name>/ --upload-file mysql-1.4.0.tgz -v
```

### PR DESCRIPTION
**Pull request by vmax**
Fix #24

### CODE REVIEW

File: helm/helm-push.bzl [hunk 1]
```diff
@@ -56,6 +64,7 @@ helm_push = rule(
       "repository_url": attr.string(mandatory = True),
       "repository_username": attr.string(mandatory = True),
       "repository_password": attr.string(mandatory = True),
+      "repo_type": attr.string(values = ["museum", "nexus"], default = "museum")
```
Comment (gutmox) [inline helm/helm-push.bzl:67]: @vmax this looks good. Could you be so kind to add some documentation about the new 'repo_type' attribute, please?

Comment (gutmox) [reply helm/helm-push.bzl:67]: Many thanks for your contribution to this repo, btw :-)

Comment (vmax) [reply helm/helm-push.bzl]: @gutmox addressed in https://github.com/masmovil/bazel-rules/pull/25/commits/16f5d117923b30527f3fa61b7b589f6cc42a3f3a

**Comment by vmax** [comment]
@gutmox thank you for accepting the PR!

**Comment by gutmox** [comment]
> @gutmox thank you for accepting the PR!

Thank you @vmax for contributing; We are really glad of sharing some of our work and seeing it's useful for others.

**Comment by lolski** [comment]
Yes this Bazel rule is really helpful! Thanks for making it.

**Pull request merged**